src/main/resources/org/jfrog/hudson/ArtifactoryBuilder/global.jelly

```diff
@@ -123,16 +124,62 @@
 
                     <f:block>
                         <div align="right">
-                            <f:repeatableDeleteButton />
+                            <f:repeatableDeleteButton/>
                         </div>
                     </f:block>
 
                 </table>
             </f:repeatable>
         </f:entry>
+        <f:block>
+            <br/>
+        </f:block>
+        <f:entry title="Pipelines server" description="Send job results to JFrog Pipelines.">
+            <table width="100%">
+                <f:entry title="Callback URL"
+                         help="/plugin/artifactory/help/ArtifactoryBuilder/help-pipelinesCbkUrl.html">
+                    <f:textbox field="pipelinesCbkUrl" value="${descriptor.pipelinesServer.cbkUrl}"/>
+                </f:entry>
+
+                <f:advanced>
+                    <f:entry>
+                        <f:entry title="Connection Timeout" field="pipelinesTimeout"
+                                 help="/plugin/artifactory/help/ArtifactoryBuilder/help-timeout.html">
+                            <f:textbox clazz="number" value="${descriptor.pipelinesServer.timeout}"/>
+                        </f:entry>
+                        <f:entry field="connectionRetry" title="Number of retries"
+                                 help="/plugin/artifactory/help/ArtifactoryBuilder/help-connectionRetry.html">
+                            <select class="setting-input" name="pipelinesConnectionRetry">
+                                <j:forEach var="r" items="${descriptor.pipelinesServer.connectionRetries}">
+                                    <f:option selected="${r==descriptor.pipelinesServer.connectionRetry}"
+                                              value="${r}">${r}
+                                    </f:option>
+                                </j:forEach>
+                            </select>
+                        </f:entry>
+                        <f:optionalBlock name="pipelinesBypassProxy" title="Bypass HTTP Proxy"
```

Comment:
Bypass HTTP Proxy --> Bypass HTTP proxy

(should also fix this for the Artifactory server configutration.)